js/researches/countLinks.js

```diff
@@ -0,0 +1,16 @@
+/** @module analyses/getLinkStatistics */
+
+var getLinks = require( "./getLinks" );
+
+/**
+ * Checks a text for anchors and returns the number found.
+ *
+ * @param {object} paper The paper object containing text, keyword and url.
+ * @returns {number} The number of links found in the text.
+ */
+module.exports = function( paper ) {
```

Comment:
The paper is always passed to a research, this is done because is research is called by the researcher and always passes the paper as an argument.